docs/admin/node-conformance.md

```diff
@@ -9,54 +9,52 @@ assignees:
 
 ## Node Conformance Test
 
-*Node conformance test* is a test framework validating whether a node meets the
-minimum requirement of Kubernetes with a set of system verification and
-functionality test. A node which passes the tests is qualified to join a
-Kubernetes cluster.
+*Node conformance test* is a containerized test framework validating whether a
+node meets the minimum requirement of Kubernetes with a set of system
+verification and functionality test. A node which passes the test is qualified
+to join a Kubernetes cluster.
 
 ## Limitations
 
 There are following limitations in the current implementation of node
-conformance test. They'll be improved in future version.
+conformance test.
 
 * Node conformance test only supports Docker as the container runtime.
-* Node conformance test doesn't validate network related system configurations
-  and functionalities.
 
 ## Prerequisite
 
 Node conformance test is used to test whether a node is ready to join a
 Kubernetes cluster, so the prerequisite is the same with a standard Kubernetes
-node. At least, the node should have properly installed:
+node. At least, the node should have installed:
 
 * Container Runtime (Docker)
 * Kubelet
 
-Node conformance test validates kernel configurations. If the kenrel module
-`configs` is built as module in your environment, it must be loaded before the
-test. (See [Caveats #3](#caveats) for more information)
-
 ## Usage
 
 ### Run Node Conformance Test
 
 * **Step 1:** Point your Kubelet to localhost `--api-servers="http://localhost:8080"`,
-because the test framework starts a local master to test Kubelet.
+because the test framework starts a local master to test Kubelet. There are some
+other Kubelet flags you may care:
+  * `--pod-cidr`: If you are using `kubenet`, you should specify an arbitrary CIDR
+    to Kubelet, for example `--pod-cidr=10.180.0.0/24`.
+  * `--cloud-provider`: If you are using `--cloud-provider=gce`, you should
+    remove the flag to run the test.
 
 * **Step 2:** Run the node conformance test with command:
 
 ```shell
-# $CONFIG_DIR is the pod manifest path of your kubelet.
+# $CONFIG_DIR is the pod manifest path of your Kubelet.
 # $LOG_DIR is the test output path.
 sudo docker run -it --rm --privileged --net=host \
-  -v /:/rootfs:ro -v /var/run:/var/run \
-  -v $CONFIG_DIR:/etc/manifest -v $LOG_DIR:/var/result \
-  gcr.io/google_containers/node-test-amd64:v0.1
+  -v /:/rootfs -v $CONFIG_DIR:$CONFIG_DIR -v $LOG_DIR:/var/result \
+  gcr.io/google_containers/node-test:0.2
 ```
 
 ### Run Node Conformance Test for Other Architectures
 
-We also build node conformance test docker images for other architectures:
+Find node conformance test docker images for other architectures:
```

Comment:
"Kubernetes also provides node conformance test docker images for other architectures:"